packages/flash-swap/test/integration/underlyingFlashUniswapV2/effects/uniswapV2Call.ts

```diff
@@ -357,53 +357,41 @@ export function shouldBehaveLikeUniswapV2Call(): void {
                     await this.contracts.uniswapV2Pair.sync();
                   });
 
-                  context("when wrong token is flash borrowed", function () {
```

Comment:
@paulrberg Just to note, removing this block brought down the test coverage.

Reply:
Oh I see. I removed it because I think it was accidentally duplicated - there is a test for the same thing a couple of blocks above.

Let me think about what we can do about it.

Reply:
Update: I think it's okay to merge this PR without the test I removed here. It's the same story in `CollateralFlashUniswapV2` - that if block is not accounted for by the tests (mainly to fend off complexity in the test files, but I am open to write the test anyway, just not in this PR).

The logic that is not tested and deserves more attention is this if block:

```solidity
if (vars.repayUnderlyingAmount > vars.seizedUnderlyingAmount)
```